tests/integration/TEC/Tickets/Admin/All_Tickets/__snapshots__/List_TableTest__test_filters__filter by discounted__0.snapshot.json

```diff
@@ -0,0 +1 @@
+null
```

Comment:
I would double check the tests producing these `null` snapshots.

Reply:
This is actually correct. I purposely made no discounted or past tickets so the tests would be blank.